src/ng/log.js

```diff
@@ -11,6 +11,9 @@
  *
  * The main purpose of this service is to simplify debugging and troubleshooting.
  *
+ * To reveal the location of the calls to `$log` in the JavaScript console,
+ * "blackbox" the Angular source in your browser.
```

Comment:
I think these links are okay. You can add both and a note that *some* browsers supprt blackboxing.